source/val/validate.cpp

```diff
@@ -243,53 +243,52 @@ spv_result_t ValidateBinaryUsingContextAndValidationState(
   }
 
   for (auto& instruction : vstate->ordered_instructions()) {
-    {
```

Comment:
Instead of removing this block, can we wrap the usageon 291 in it's own block? I'd like to keep the non-const version of instruction as contained as possible.